src/providers/definition/controllerDefinitionProvider.ts

```diff
@@ -12,6 +12,12 @@ export class ControllerDefinitionProvider extends BaseDefinitionProvider {
 				return [ path.join(project.filePath, 'app', 'lib', `${value}.js`) ];
 			}
 		},
+		{ // ES6 import from (/lib) name
+                	regExp: /import (?:[-a-zA-Z0-9-_/\s]*)['"]([-a-zA-Z0-9-_/]*)$/,
```

Comment:
<!--
  1 failure:  src/providers/def...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: src/providers/definition/controllerDefinitionProvider.ts;
  Line: 16;
-->
- :no_entry_sign: src/providers/definition/controllerDefinitionProvider.ts line 16 – Mixed spaces and tabs. (no-mixed-spaces-and-tabs)